subprojects/common/kotlin-ast-parser/src/main/kotlin/com/avito/instrumentation/impact/KotlinClassesFinder.kt

```diff
@@ -0,0 +1,10 @@
+package com.avito.instrumentation.impact
+
+import java.io.File
+
+interface KotlinClassesFinder {
+
+    fun find(file: File): List<String>
```

Comment:
How about `Set<..>`?
Do we have a case with duplicates?

Reply:
i changed it to Sequence :)